config/serverSettings.js

```diff
@@ -24,8 +24,9 @@ const devSessionKey = devDefaults('node-web.sid')
 const devSessionUseRedis = devDefaults(true)
 const devRedis = devDefaults('redis://localhost:6379/')
 const devKoppsApi = devDefaults('https://api-r.referens.sys.kth.se/api/kopps/v2/?defaultTimeout=60000')
+const devProgramSyallabusPDFURL = devDefaults('https://integral-api.sys.kth.se/test/PDFRenderFunction')
+const devServicePublish = '/student/kurser'
```

Comment:
I think that if this is named **dev**ServicePublish it should also be wrapped in devDefaults.

So this should be changed to
```suggestion
const devServicePublish = devDefaults( '/student/kurser')
```
or  remove **dev** if its not supposed to be dev only(maybe call it `servicePublish` or `defaultServicePublish`)
```suggestion
const servicePublish = '/student/kurser'
```